presto-main/src/main/java/com/facebook/presto/memory/MemoryManagerConfig.java

```diff
@@ -39,6 +39,7 @@
     private DataSize softMaxQueryTotalMemory;
     private String lowMemoryKillerPolicy = LowMemoryKillerPolicy.NONE;
     private Duration killOnOutOfMemoryDelay = new Duration(5, MINUTES);
+    private boolean trackMemoryInTableFinishOperator;
```

Comment:
nit: `tableFinishOperatorMemoryTrackingEnabled` (also get/set and the property name)